lib/index.js

```diff
@@ -83,9 +84,20 @@ Mixpanel.prototype.identify = function(identify, fn){
     verbose: 1        // make sure that we get a valid response
   };
 
+  var traits = formatTraits(identify);
+
+  if (!this.settings.setAllTraitsByDefault) {
```

Comment:
This is added to mirror the client-side integration. This is a breaking change in that anyone who doesn't have `setAllTraitsByDefault` enabled will no longer receive every trait as a people property. Consequently, not having this would be breaking for people who are unbundling as they would suddenly start getting all traits saved as people properties.

This will be released on a new version that will only be used by people who are unbundling mixpanel, and that will include a big warning that lists this as a behavior change.